### DISCUSSION TITLE
When opening a filtered down issue list through a link, the mobile app ignores the supplied filter and just takes the last used.

### DISCUSSION THREAD

**Discussion by jessehouwing** [community/community#32336] [Mobile]
When i am sent a filtered issue list through email or in a releasenote and open it on my phone the mobile app pops up, but it completely ignores the filter in the link:

https://github.com/sdo/repo/issues?q=is%3Aissue+label%3APSU+label%3Av3.3.3+

Instead I get the last configured filter from previously working int the app.

**Comment by eliperkins** [comment]
Hey @jessehouwing , thanks for this feedback!

I can reproduce this issue as well, so you're absolutely right that this is a bug. I've put together your notes here in an internal issue on our repo to get this fixed. I'll let you know when this bug gets addressed. 🙇 

In the meantime, a couple of workarounds are to:

- Use GitHub.com within your browser to access known links like this, or
- Use [Shortcuts on GitHub Mobile](https://github.blog/changelog/2022-01-13-introducing-shortcuts-on-github-mobile/) for common repo issues links like this one

Let me know if this helps for now! We hope to have this bug fixed soon.
